scripts/check-health.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# assert on the health of a running container
+IS_SYNCING=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.isSyncing')
+SHOULD_HAVE_PEERS=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.shouldHavePeers')
+PEERS=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.peers')
+
+if [ "$IS_SYNCING" != "true" ]; then
+    echo "FAIL"
+    exit 1
+fi
+
+if [ "$SHOULD_HAVE_PEERS" != "true" ]; then
```

Comment:
This is only true if you have specified some `--bootnodes` when you started the node. If you start a node without bootnodes (for instance if you're running a local network or you are a bootnode yourself) this will be false.